influxdb/client_test.go

```diff
@@ -213,10 +219,14 @@ func TestClient_WriteLarge(t *testing.T) {
 			t.Errorf("expected gzip Content-Encoding but got %s", r.Header.Get("Content-Encoding"))
 		}
 		if string(bod) != expected.String() {
-			t.Errorf("unexpected send, expected '%s', got '%s'", expected, string(bod))
+			t.Errorf("unexpected send:\n%s",
```

Comment:
Does `Errorf` halt the test? (Is there a chance the `wg.Done()` below will never be called?)

Reply:
`Errorf` does not halt the test, just logs and marks the test as having failed.